scripts/publish_external.sh

```diff
@@ -1,14 +1,8 @@
 # undo permissions change on scripts/publish_external.sh
 git checkout -- scripts
 
-# Use newer, faster npm
-npm install -g npm@5.0.3
-
-# Install required dependencies
-npm install -g lerna
-
-if ! lerna bootstrap; then
-  echo "lerna bootstrap failed! Exiting..."
+if ! yarn install; then
```

Comment:
This fails on Bacon, I created a JIRA for eng services for a similar problem in okta-signin-widget https://oktainc.atlassian.net/browse/OKTA-190445 in case we cannot find a solution.